src/main/resources/jseont_sitemap/xml/sitemap.index.jsp

```diff
@@ -0,0 +1,64 @@
+<%@ taglib prefix="jcr" uri="http://www.jahia.org/tags/jcr" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="utility" uri="http://www.jahia.org/tags/utilityLib" %>
+<%@ taglib prefix="template" uri="http://www.jahia.org/tags/templateLib" %>
+<%@ taglib prefix="functions" uri="http://www.jahia.org/tags/functions" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+<%@ taglib prefix="query" uri="http://www.jahia.org/tags/queryLib" %>
+
+<%--@elvariable id="currentNode" type="org.jahia.services.content.JCRNodeWrapper"--%>
+<%--@elvariable id="currentResource" type="org.jahia.services.render.Resource"--%>
+<%--@elvariable id="flowRequestContext" type="org.springframework.webflow.execution.RequestContext"--%>
+<%--@elvariable id="out" type="java.io.PrintWriter"--%>
+<%--@elvariable id="renderContext" type="org.jahia.services.render.RenderContext"--%>
+<%--@elvariable id="script" type="org.jahia.services.render.scripting.Script"--%>
+<%--@elvariable id="scriptInfo" type="java.lang.String"--%>
+<%--@elvariable id="url" type="org.jahia.services.render.URLGenerator"--%>
+<%--@elvariable id="workspace" type="java.lang.String"--%>
+
+<c:set target="${renderContext}" property="contentType" value="text/xml;charset=UTF-8"/>
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+        <jsp:include page="../../common/serverUrl.jsp"/>
+        <c:set var="siteMapPath" value="${fn:replace(currentNode.url, '.html', '')}" />
+        <sitemap>
+                <loc>${serverUrl}<c:url value="${siteMapPath}.xml"/></loc>
+        </sitemap>
+        <sitemap>
```

Comment:
I know it was part of the (outdated) specs but no point adding blank sitemaps for now. let's comment this out for the custom, images and resources sitemaps

Reply:
Ok, I will comment it out for now